geniusyield-market-maker/src/GeniusYield/MarketMaker/Prices/Taptools.hs

```diff
@@ -0,0 +1,95 @@
+module GeniusYield.MarketMaker.Prices.Taptools where
+
+import           Data.Aeson
+import           Data.Aeson.Types                 (typeMismatch)
+import qualified Data.Map                         as Map
+import           Data.Proxy
+import           Data.Text                        (Text)
+import           Data.Time.Clock.POSIX            (POSIXTime)
+import           GeniusYield.GYConfig             (Confidential (..))
+import           GeniusYield.Imports              (Generic)
+import           GeniusYield.MarketMaker.Utils
+import           Network.HTTP.Client              (newManager, Manager)
+import           Network.HTTP.Client.TLS          (tlsManagerSettings)
+import           Servant.API
+import           Servant.Client
+
+
+-- | Taptools time resolution for OHLC Candles
+data TtResolution = TtRes3m | TtRes5m | TtRes15m | TtRes30m | TtRes1h | TtRes2h | TtRes4h | TtRes12h
+                  | TtRes1d | TtRes3d | TtRes1w  | TtRes1mo
```

Comment:
Let's match with TapTools specification for it's provider: `TtRes1mo` -> `TtRes1M`.

Reply:
Done.